index.html

```diff
@@ -0,0 +1,289 @@
+                                </div>
+                                <div id="elipse3">
+                                    <img src="Ellipse3.png" alt="">
+                                </div>
+                                <div id="rectangle4">
+                                    <img src="Rectangle4.png" alt="">
+                                </div>
+                                <div id="elipse8">
+                                    <img src="Ellipse8.png" alt="">
+                                </div>
+                            </section>
+                            <div class="toolbar2">
+                                <img src="Union.png" alt="Menu">
+                            </div>
+                        </nav> 
+                    
+                    </div>
+                </header>
+                <section id="primary">
+                    <div class="name">
+                        <h1>I'm Kelvin Glad to see you!</h1>
+                    </div>
+                    <div class="text">
+                        <p>I'm a software developer! I can help you build a product,
+                            feature or website. Look through someof my work and experience!
+                            If you like what you see and have a project you need coded,
+                            don't hesitate to contact me.
+                        </p>
+                    </div>
+                    <nav id="lets-connect">
+                        <h5>LET'S CONNECT</h5>
+                    </nav>
+                    <div>
+                        <ul class="icons">
+                            <li><img src="Vector.png" alt=""></li>
+                            <li><img src="Linkedin.png" alt=""></li>
+                            <li><img src="Vector1.png" alt=""></li>
+                            <li><img src="Vector3.png" alt=""></li>
+                        </ul>
+                    </div>
+                    <section class="bottom-images">
+                        <div id="elipse12">
+                            <img src="Ellipse12.png" alt="">
+                        </div>
+                        <div id="elipse13">
+                            <img src="Ellipse13.png" alt="">
+                        </div>
+                        <div id="elipse15">
+                            <img src="Ellipse15.png" alt="">
+                        </div>
+                        <div id="elipse4">
+                            <img src="Ellipse4.png" alt="">
+                        </div>
+                        <div id="rectangle5">
+                            <img src="Rectangle5.png" alt="">
+                        </div>
+                    </section>
+                </section>
+            </div>
+    </article>
+        
+        <div class="work-card">
+            <div class="work1">
+                <img src="Snapshoot.png" alt="">
+               <div id="block1">
```

Comment:
- [ ] I think it would be better to use [Semantic HTML5](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) tags to follow the best practice, so I suggest using `<section>` tag instead of  `<div>` in the work section. 👌